### PR TITLE
4.next: use `->hasValue()` instead of `->has()`

### PR DESCRIPTION
**Pull request by LordSimal**
Refs: https://github.com/cakephp/bake/issues/931

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: en/appendices/4-5-migration-guide.rst [hunk 1]
```diff
@@ -80,6 +80,9 @@ Log
 ORM
 ---
 
+- In CakePHP 5 ``EntityTrait::has()`` will return true when an attribute exists
+  and is set to null. To keep the old behavior it is recommended to
+  change ``EntityTrait::has()`` to ``EntityTrait::hasValue()``.
```
Comment (othercorey) [inline en/appendices/4-5-migration-guide.rst]: I don't think this is true. hasValue() has completely different behavior than has().

Comment (LordSimal) [reply en/appendices/4-5-migration-guide.rst]: Sure it ain't a 1:1 replacement. What would be the recommended replacement to not get warnings in the template for already baked templates? Keeping it as it is will result in warnings being printed.

Comment (othercorey) [reply en/appendices/4-5-migration-guide.rst]: We just can't say "To keep the old behavior".  Need to explain there's an alternative that returns false on null but has other behavior too.

Comment (LordSimal) [reply en/appendices/4-5-migration-guide.rst]: Thats true. I tried to reword it.

**Comment by othercorey** [comment]
Looks ok to me now

File: en/appendices/4-5-migration-guide.rst [hunk 2]
```diff
@@ -80,6 +80,10 @@ Log
 ORM
 ---
 
+- In CakePHP 5 ``EntityTrait::has()`` will return true when an attribute exists
+  and is set to null. Depending on your application this can lead to unexpected
+  behavior with your already present code. Use ``EntityTrait::hasValue()`` to
```
Comment (markstory) [inline en/appendices/4-5-migration-guide.rst]:
```suggestion
  behavior with your existing code. Use ``EntityTrait::hasValue()`` to
```

**Pull request merged**